desparchado/settings/base.py

```
@@ -5,6 +5,7 @@
 from django.core.exceptions import ImproperlyConfigured
 
 from dotenv import load_dotenv
+from tests.testapp.testapp.settings import GOOGLE_MAP_API_KEY
```

Ruff (0.8.2)

`tests.testapp.testapp.settings.GOOGLE_MAP_API_KEY` imported but unused

Remove unused import: `tests.testapp.testapp.settings.GOOGLE_MAP_API_KEY`

(F401)